src/lib/final_report.ml

```diff
@@ -72,19 +73,26 @@ module To_json = struct
       ?seq2hla
       ?stringtie
       ?bedfile
+      ?kallisto
       ~metadata
       run_name =
-    fun ~var_count ->
+   fun ~var_count ->
       let args =
-        let opt n o =
-          Option.value_map ~default:[] o ~f:(fun v -> [n, v ~var_count]) in
+        let opt n =
+          Option.value_map ~default:[] ~f:(fun o -> [n, o ~var_count])
+        in
+        let named_map ?(nf=(fun n -> n)) = 
```

Comment:
Also might factor some of these out if they're being used across functions/modules.